app/assets/stylesheets/webserver/main.css.scss

```diff
@@ -1,254 +1,255 @@
 html {
-    position: relative;
```

Comment:
The file has been re-formated to tabsize=2, which messes up the diff. Is it difficult to return the file to 2 spaces?

Reply:
To know what the formatter is doing:vscode -> toggle panel -> output -> prettier.

The file is outside the covid project so probably vscode is using its default tabsize, simply chaning that to 4 should do the trick.